grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -897,6 +897,17 @@ def section_streaming_actors(outer_panels):
                         ),
                     ],
                 ),
+
+                panels.timeseries_bytes(
+                    "Materialized View Memory Usage",
+                    "Materialzed View Memory Usage",
+                    [
+                        panels.target(
+                            f"sum(stream_memory_usage * on(table_id, actor_id) group_left(materialized_view_id) table_info) by (materialized_view_id)",
```

Comment:
`stream_memory_usage` should be replaced by `{metric('stream_memory_usage')}` so that the metric fn will be called to add extra filters required by the cloud dashboard.

Reply:
Got it, thanks! Fixed.